index.md

```diff
@@ -322,43 +322,47 @@ puts "height = #{g.logical_screen.image_height}"</code></pre>
     </ul>
     <div class="tab-content" style="margin-top: 20px; min-height: 300px">
-      <div role="tabpanel" class="tab-pane active" id="download-linux-deb">
+      <div role="tabpanel" class="tab-pane active" id="download-universal">
         <div class="row">
           <div class="col-sm-9">
-            <p>The stable <code>kaitai-struct-compiler</code> versions are currently uploaded to <a href="https://github.com/kaitai-io/kaitai_struct_compiler/releases" target="_blank" rel="noopener">https://github.com/kaitai-io/kaitai_struct_compiler/releases</a> (see <a href="#bintray-sunset-callout">box above</a>). Just download the <code>.deb</code> package and install it:
+            <p>"Universal" builds are downloadable as
+              a .zip file that includes all the required
+              .jar files bundled and launcher scripts
+              for Linux / macOS / Windows systems. No
+              installation required, one can just unpack
+              and run it.
             </p>
-            <pre>curl -LO https://github.com/kaitai-io/kaitai_struct_compiler/releases/download/0.9/kaitai-struct-compiler_0.9_all.deb
-sudo apt-get install ./kaitai-struct-compiler_0.9_all.deb</pre>
+            <p><a class="btn btn-success" href="https://github.com/kaitai-io/kaitai_struct_compiler/releases/download/0.9/kaitai-struct-compiler-0.9.zip"><i class="fa fa-download"></i> Download</a> — stable v0.9, 9.5 MiB</p>
+            <p><a class="btn btn-default disabled" href="#0"><i class="fa fa-download"></i> Download</a> — <del class="text-danger">latest development (unstable) build</del> — currently not available (see <a href="#bintray-sunset-callout">box above</a>)</p>
           </div>
           <div class="col-sm-3">
             <h4>Requirements</h4>
             <ul>
-              <li>.deb-based Linux distribution (Debian, Ubuntu, etc)</li>
+              <li>Linux / macOS / Windows system</li>
+              <li>a working <a href="http://java.com/download/">Java Runtime Environment</a> installation</li>
             </ul>
           </div>
         </div>
       </div>
-      <div role="tabpanel" class="tab-pane" id="download-mac-homebrew">
+      <div role="tabpanel" class="tab-pane" id="download-linux-deb">
         <div class="row">
           <div class="col-sm-9">
-            <p>There is a
-              <a href="https://formulae.brew.sh/formula/kaitai-struct-compiler">Homebrew formula</a>
-              that you can use to install <code>kaitai-struct-compiler</code>:
+            <p>The stable <code>kaitai-struct-compiler</code> versions are currently uploaded to <a href="https://github.com/kaitai-io/kaitai_struct_compiler/releases" target="_blank" rel="noopener">https://github.com/kaitai-io/kaitai_struct_compiler/releases</a> (see <a href="#bintray-sunset-callout">box above</a>). Just download the <code>.deb</code> package and install it:
```

Comment:
This may be a useful phrase for all options (the link to the GitHub Releases page is not present elsewhere) - the **Linux .deb** tab used to be selected by default, so anyone could read it and go to the GH Releases page if needed.

But it's no longer the default option, so it would be hidden in the **Linux .deb** tab now and I'm not sure everyone would be able to find it.